variables.tf

```diff
@@ -1,20 +1,22 @@
+// ----------------------------------------------------------------------------
+// Required Variables
+// ----------------------------------------------------------------------------
 variable "region" {
-  type = string
+  type    = string
```

Comment:
on the top-level variables we should always add a description so that the generated markdown table is complete

Reply:
+1